packages/react/src/table/TableScroller.tsx

```diff
@@ -230,6 +232,26 @@ export function TableScroller({ children }: TableScrollerProps) {
 				tabIndex={0}
 			>
 				{children}
+				<Shadow
+					edge="left"
+					height={scrollerRef?.current?.offsetHeight || 0}
+					isVisible={Boolean(
+						thumbWidthRatio < 1 &&
+							scrollerRef?.current?.scrollLeft &&
+							scrollerRef?.current?.scrollLeft > 0
```

Comment:
Shouldn't need the `?` on this line since they're checked on the line above.

Reply:
Thanks, fixed